aiohttp/web_protocol.py

```diff
@@ -402,6 +404,7 @@ def _process_keepalive(self) -> None:
 
             message, payload = self._messages.popleft()
 
+            now = None
```

Comment:
I think we can always use unconditional `now = loop.time()` here.
The call is relative cheap in comparison to all other method's logic.
Do you have evidence that conditional initialization speeds up things?

Reply:
ok

Reply:
fixed, I had no evidence of the speed, I just kept it the same as before for consistency.

I've also renamed the variable since `now` was a dumb name.